td4m3/src/main/res/values/dimens.xml

```diff
@@ -8,4 +8,5 @@
 
     <dimen name="half_pie_chart_height">200dp</dimen>
     <dimen name="card_divider_spacing">5dp</dimen>
+    <dimen name="text_20_sp">20sp</dimen>
```

Comment:
This goes in fonts.xml, not in dimens.xml

Reply:
Done.